### ISSUE TITLE
Enhance  MD description and instruction

### ISSUE THREAD

**Issue by jecalderon** [bobthechemist/mma-thermal-camera#1]
This project could well use instruction and description contents in the .md file.  Must Mathematica users do not know how to upload packages. Also need  definition of the functions descriptions.

**Comment by bobthechemist** [comment]
I agree.  This was a one-afternoon project and I posted it here to avoid losing the code.  It's proof-of-concept and not quite ready for prime time.  Thanks for checking it out.
